frontend/src/views/groups/GroupsView.tsx

```diff
@@ -2,11 +2,12 @@ import { useQuery } from "@tanstack/react-query";
 import { Table } from "../../components/table";
 import { ME_URL, getGroupLeaderboardUrl } from "../../helpers/api";
 import axios, { AxiosResponse } from "axios";
-import { Group, User } from "../../helpers/types";
+import {Group, GroupUser, User} from "../../helpers/types";
```

Comment:
Use prettier, please for the love of god.

Reply:
yep